workflows/sample_analysis/sample_analysis.wdl

```diff
@@ -202,7 +202,7 @@ workflow sample_analysis {
 		IndexData phased_sv_vcf = hiphase.phased_vcfs[1]
 		File hiphase_stats = hiphase.hiphase_stats
 		File hiphase_blocks = hiphase.hiphase_blocks
-		File hiphase_haplotags = hiphase.hiphase_haplotags
+		File? hiphase_haplotags = hiphase.hiphase_haplotags
```

Comment:
Since you're sure you'll produce output here in the sample analysis, you can make this file non-optional if you want:
`File hiphase_haplotags = select_first([hiphase.hiphase_haplotags])`

Stylistic choice more than anything, it won't fail since you know this will have output here (or should). You'd need to change `Array[File?]` -> `Array[File]` in main, also.